mavros_extras/src/plugins/mount_control.cpp

```diff
@@ -81,6 +83,27 @@ class MountControlPlugin : public plugin::PluginBase {
 		mount_orientation_pub.publish(quaternion_msg);
 	}
 
+	/**
+	 * @brief Publish the mount status
+	 *
+	 * @param msg   the mavlink message
+	 * @param ms	received MountStatus msg
+	 */
+	void handle_mount_status(const mavlink::mavlink_message_t *, mavlink::ardupilotmega::msg::MOUNT_STATUS &ms)
+	{
+		geometry_msgs::Vector3Stamped publish_msg;
+
+		publish_msg.header.stamp = ros::Time::now();
+
+		publish_msg.header.frame_id = std::to_string(ms.target_component);
+
+		publish_msg.vector.x = ms.pointing_a;
+		publish_msg.vector.y = ms.pointing_b;
+		publish_msg.vector.z = ms.pointing_c;
+
+		mount_status_pub.publish(publish_msg);
```

Comment:
Maybe it's better to publish quaternion as well as rpy? Like in the code above.

Reply:
Do you have any suggestion for topic's name?

Reply:
Orientation. II suppose that APM do not send MOUNT_ORIENTATION, right?
So it's ok to publish both euler angles and quaternion.